lib/rules/jest/no-if.js

```diff
@@ -43,6 +51,16 @@ module.exports = {
           node,
         });
       },
+      ConditionalExpression(node) {
```

Comment:
One other change I'd really like to see (I think I opened a separate issue for it) is not to complain about this in nested functions. IMO, this is totally fine:

```tsx
it('works', () => {
  const graphQL = createGraphQL({variables} => variables.first ? listOf(variables.first) : listOf(50));
});
```